packages/@expo/cli/src/utils/ip.ts

```diff
@@ -1,5 +1,81 @@
 import internalIp from 'internal-ip';
+import { spawnSync } from 'node:child_process';
+import { isIPv4 } from 'node:net';
+import { networkInterfaces } from 'node:os';
+import { dirname, basename } from 'node:path';
+
+/** Gets a route address by opening a UDP socket to a publicly routed address.
+ * @privateRemarks
+ * This is wrapped in `spawnSync` since the original `getIpAddress` utility exported
+ * in this module is used synchronosly. An appropriate timeout has been set and UDP
+ * ports don't send a message when opened.
+ */
+function getRouteAddress(): string | null {
+  const command = process.platform === 'win32' ? basename(process.execPath) : process.execPath;
```

Comment:
Wait, shouldn't the `process.execPath` be correctly adjusted on Windows already?

Reply:
actually, maybe this is related to `windowsVerbatimArguments`. Disabling that caused issues with prior arguments I passed, but maybe that can be flipped again 🤔 I'll have a quick check

Reply:
ha, yea, that fixes it. I'll flip it back